proof-of-spacetime.md

```diff
@@ -0,0 +1,184 @@
+## Proof-of-Spacetime
+
+This document describes Rational-PoSt, the Proof-of-Spacetime used in Filecoin.
+
+## Rational PoSt
+
+### Definitions
+
+- **POST_PROVING_PERIOD**: The time interval in which a PoSt has to be submitted.
+- **POST_CHALLENGE_TIME**: The time offset at which the actual work of generating the PoSt should be started. This is some delta before the end of the `Proving Period`, and as such less then a single `Proving Period`.
```

Comment:
this should be backed with some research on:
- how long it takes to get a message in other currencies, for now, we can set to 2h
- how long it takes to generate a proof

add this as a todo note or as a design note ^